src/orchard/java.clj

```diff
@@ -58,17 +58,24 @@
                (io/file parent "lib" f)]]
     (->> paths (filter #(.canRead ^File %)) first io/as-url)))
 
+(def fetch-java-sources-via-dynapath?
```

Comment:
I think "fetch" is not the right word here, as we're just adding them to the classpath if they are already present locally. Probably "add/load" would be better.